provider/datagen/src/bin/datagen.rs

```diff
@@ -375,11 +377,6 @@ fn main() -> eyre::Result<()> {
         _ => unreachable!(),
     };
 
-    icu_datagen::datagen(
-        selected_locales.as_deref(),
-        &selected_keys,
-        &source_data,
-        vec![out],
-    )
-    .map_err(eyre::ErrReport::from)
+    icu_datagen::datagen(Some(&locales), &selected_keys, &source_data, vec![out])
```

Comment:
Issue: I still think there should be a way from the CLI to specify the "lack of a locale filter"

Reply:
Is this a different discussion than the previous thread?

I can make "full" set this to `None` instead of `Some(source.locales(&[Basic, Moderate, Modern)?)`. This might actually be different behaviour though: if CLDR and ICU export are not "synced", ICU export might contain newer locales that are not in `coverageLevels.json`. I'm not sure if including them then would make sense.

Reply:
It's the same as the previous thread, sorry. Yes, I'm just asking that the CLI have a way to set the locale filter to `None`. Making `--locales full` do this behavior is okay with me. If someone really wants the strict coverage levels behavior, they can do `--locales basic moderate modern`.